docs/quickstart/readme.md

```diff
@@ -0,0 +1,310 @@
+</details>
+
+Now, on the upper left of Blender, click on `Add` -> `Mesh` -> `Cube`. This will be our player. We will now add the `Player` component to it. Go to the Blenvy menu we opened earlier. Select the icon in its upper left corner. This is where components are managed.
+
+<details>
+<summary>The component manager</summary>
+<img src="img/component_manager.png" width = 50%/>
+</details>
+
+If your component manager says "Select an object to edit its components", make sure you have the *Cube* we just created selected by clicking on it in the 3D viewport.
+
+Click on `Components`. If it says "No results found", we need to explicitly reload the registry. In the Blenvy menu, click on the gear icon (the fourth one from the left). In the submenu, click on the second icon. Your menu should now look like this:
+
+<details>
+<summary>The registry settings</summary>
+<img src="img/registry_settings.png" width = 50%/>
+</details>
+
+Click on `reload registry` to reload it manually.
+Go back to the component manager and you should now be greeted by a whole bunch of components:
+
+<details>
+<summary>The registry settings</summary>
+<img src="img/lots_of_components.png" width = 50%/>
+</details>
+
+Type in `Player` and click on `my_game::Player`.
+
+> [!TIP]
+> If this does not show up, you have forgotten to call `register_type` with it.
+
+Now click `Add`. You can now set all the player's fields in a neat Blender UI. Note that while we are only using `f32` here, you can use any kind of field, even complicated nested enums or structs.
+
+<details>
+<summary>The player component in Blender</summary>
+<img src="img/player_component.png" width = 50%/>
+</details>
+
+Congratulations, you have just created your first blueprint.
+
+## Compose the world
+
+Let's populate our world now. Switch back to the `World` scene in Blender as described before.
+
+We will add the following kinds of objects to our world:
+
+- Some basic ground, which we will create directly in the world
+- A camera
+- A light
+- The player character, which will be a blueprint instance
+
+First, let's add a camera through `Add` -> `Camera`. Blender will make the camera face whatever direction the viewport is aimed at, which will probably be completely arbitrary.
+Move the camera a bit so that it faces the world origin and is rotated the right way.
+Pressing `0` on your numpad will show you a preview of what the camera sees.
+You can just copy our transform if you like:
+
+- Location:
+  - X: `0`
+  - Y: `-10`
+  - Z: `2`
+- Rotation
+  - X: `80`
+  - Y: `0`
+  - Z: `0`
+- Scale
+  - X: `1`
+  - Y: `1`
+  - Z: `1`
+
+  All transforms we are using assume that you are using the `XYZ Euler` mode for rotation.
+
+<details>
+<summary>The camera transform in Blender</summary>
+<img src="img/camera_transform.png" width = 50%/>
+</details>
+
+Now, let's add a ground with `Add` -> `Mesh` -> `Cube`. Again scale and move it so that it lies neatly under the camera. Our transform looks like this:
```

Comment:
is there a specific reason (collider? ) for not using a plane (add -> Mesh -> Plane) instead ?

Reply:
Exactly. Colliders need some thickness, which one will notice when adding them, as there is no `Plane` collider. I added a cube to make the experience as smooth as possible without some weird asterisks.